src/applications/ivc-champva/10-7959f-1/containers/App.jsx

```diff
@@ -25,32 +25,38 @@ export default function App({ location, children }) {
   ];
   const bcString = JSON.stringify(breadcrumbList);
   return (
-    <div className="vads-l-grid-container large-screen:vads-u-padding-x--0">
-      <Toggler toggleName={Toggler.TOGGLE_NAMES.form107959F1}>
-        <Toggler.Enabled>
-          <va-breadcrumbs breadcrumb-list={bcString} />
-          <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
-            <DowntimeNotification
-              appTitle={`CHAMPVA Form ${formConfig.formId}`}
-              dependencies={[externalServices.pega]}
+    <>
+      <meta content="noindex" />
```

Comment:
Will this `<meta>` tag take effect if it's not wrapped in the main document's `<head>` tag? We may need to run a `useEffect` function that just does a `document.querySelector('head')` and appends this to the actual meta tag. But also, I'm not 100% sure that's how it works... Just a thought.

Reply:
Per the [article](https://www.mariehaynes.com/noindex-tag-in-body-of-page/) @rmessina1010 linked in slack, looks like this will behave the way we want it to.